hub-saml-test-utils/src/main/java/uk/gov/ida/saml/hub/domain/HubEidasAttributeQueryRequest.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.ida.saml.hub.domain;
```

Comment:
Why do we have a copy of this in saml-test-utils (as well as the one in hub saml)?

Reply:
I cannot find this class in saml-test-utils. Classes in saml-test-utils cannot be used for production and are used for testing only.